File: CFOnlineShop/CFOnlineShop/ViewController/CFShoppingCartController/CFShoppingCartCell1.m
//
//  CFShoppingCartCell1.m
//  CFOnlineShop
//
//  Created by chenfeng on 2018/7/25.
//  Copyright © 2018年 chenfeng. All rights reserved.
//

#import "CFShoppingCartCell1.h"

@implementation CFShoppingCartCell1

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        
        _imageView = [[UIImageView alloc] initWithFrame:CGRectMake(15, 10, 60, 60)];
        _imageView.contentMode = UIViewContentModeScaleAspectFit;
        [self.contentView addSubview:_imageView];
        
        _titleStr = [[UILabel alloc] initWithFrame:CGRectMake(MaxX(_imageView) + 15, 15, 150, 20)];
        _titleStr.font = SYSTEMFONT(14);
        _titleStr.textColor = KDarkTextColor;
        [self.contentView addSubview:_titleStr];
        
    }
    
    return self;
}

@end

File: SwiftOnlineShopDemo/SwiftOnlineShopDemo/ViewController/CFShoppingCartController/view/CFShoppingCartCell1.swift
//
//  CFShoppingCartCell1.swift
//  SwiftOnlineShopDemo
//
//  Created by chenfeng on 2019/4/29.
//  Copyright © 2019年 chenfeng. All rights reserved.
//

import UIKit

class CFShoppingCartCell1: CFEditCollectionCell {
    
    var imageView: UIImageView!
    var titleStr: UILabel!
    
    override init(frame: CGRect) {
        
        super.init(frame: frame)
        
        self.layer.masksToBounds = true
        self.layer.cornerRadius = 4
        self.backgroundColor = UIColor.white
        
        imageView = UIImageView.init(frame: CGRect(x: 15, y: 10, width: 60, height: 60))
        imageView.contentMode = .scaleAspectFit
        self.contentView.addSubview(imageView)
        
        titleStr = UILabel.init(frame: CGRect(x: imageView.mj_x + imageView.mj_w + 15, y: 15, width: 150, height: 20))
        titleStr.font = UIFont.systemFont(ofSize: 14)
        titleStr.textColor = UIColor.black
        self.contentView.addSubview(titleStr)
        
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
}
